tests/test_plt.py

```diff
@@ -27,7 +29,14 @@
 
 def check_plt_entries(filename):
     real_filename = os.path.join(TESTS_BASE, 'tests', filename)
-    ld = cle.Loader(real_filename, auto_load_libs=False, main_opts={'base_addr': 0})
+    try:
+        ld = cle.Loader(real_filename, auto_load_libs=False, main_opts={'base_addr': 0})
+    except CLECompatibilityError:
+        # Currently, CLE rely on features of `pyelftools` that haven't been published yet:
+        # Code loading MIPS binaries using `pyelftools` 0.26 and downwards will fail.
+        # See https://github.com/angr/cle/issues/255 for more details.
+        # So, let's shortcut those cases.
```

Comment:
Let's throw an exception with the above text so
- If someone wants to run this test, they know that should install the expected version of pyelftools (which is the one from GitHub master)
- This test case does not get disabled when our CI setup changes (unexpectedly) and installs pyelftools < 0.26

Reply:
I will remove this `try/catch`: there is already a check raising a meaningful error if the compatibility is broken.